README.md

```diff
@@ -1,36 +1,25 @@
-# Address Book (Level 4)
+# main
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
-[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=se-edu/addressbook-level4&amp;utm_campaign=Badge_Grade)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-F11-B2/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-F11-B2/main)
+[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/main)
+[![Coverage Status](https://coveralls.io/repos/github/CS2103JAN2017-F11-B2/main/badge.svg?branch=master)](https://coveralls.io/github/CS2103JAN2017-F11-B2/main?branch=master)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/main?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CS2103JAN2017-F11-B2/main&amp;utm_campaign=Badge_Grade)
 
-<img src="docs/images/Ui.png" width="600"><br>
-
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using
+* This is a desktop Task Manager application. It has a GUI but most of the user interactions happen using
   a CLI (Command Line Interface).
 * It is a Java sample application intended for students learning Software Engineering while using Java as
   the main programming language.
 * It is **written in OOP fashion**. It provides a **reasonably well-written** code example that is
   **significantly bigger** (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from [level 3](https://github.com/se-edu/addressbook-level3):
-    * A more sophisticated GUI that includes a list panel and an in-built Browser.
-    * More test cases, including automated GUI testing.
-    * Support for *Build Automation* using Gradle and for *Continuous Integration* using Travis CI.
-
 
 #### Site Map
 * [User Guide](docs/UserGuide.md)
 * [Developer Guide](docs/DeveloperGuide.md)
 * [Learning Outcomes](docs/LearningOutcomes.md)
-* [About Us](docs/AboutUs.md)
```

Comment:
You are supposed to give `About Us` page. Refer to the module website for more information.